src/v2/api/controllers/UserController.ts

```diff
@@ -462,28 +476,69 @@ export class UserController {
   @ApiParam({
     name: 'userId',
     type: String,
+    description: 'Id of a user to delete contact'
   })
   @ApiParam({
     name: 'contactId',
     type: String,
+    description: 'Id of a contact to delete',
+  })
+  @ApiEndpoint({
+    summary: 'Request captain or admin to delete the contact',
+    permissions: PERMISSION.USERS_$USERID_CONTACTS_DELETE,
   })
-  @Access(PERMISSION.USERS_$USERID_CONTACTS_DELETE)
   @Delete('/:userId/contacts/:contactId')
   deleteContact (
-    @Param(ContactByUserIdPipe) params,
+    @Param(ContactByUserIdPipe) params: { userId: string, contactId: string },
   ) {
     return this.userService.deleteContact(params.userId, params.contactId);
   }
 
-  @Access(PERMISSION.STUDENTS_DELETE)
+  @ApiOkResponse()
+  @ApiParam({
+    name: 'userId',
+    required: true,
+    description: 'Id of a user to delete',
+  })
+  @ApiEndpoint({
+    summary: 'Request captain or admin to delete a student',
+    permissions: PERMISSION.STUDENTS_DELETE,
+  })
   @Delete('/:userId/student')
   deleteStudent (
     @Param('userId', UserByIdPipe) userId: string,
   ) {
     return this.userService.deleteStudent(userId);
   }
 
-  @Access(PERMISSION.USERS_$USERID_STUDENT_UPDATE)
+  @ApiOkResponse({
+    type: OrdinaryStudentResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found
+      
+    InvalidBodyException:
+      First name is not correct (A-Я(укр.)\\-\' )
+      First name is too short (min 2)
+      First name is too long (max 40)
+      Last name of an updated student
```

Comment:
remove "Last name of an updated student" and unnecessary backslashes (fix your linter)